### ISSUE TITLE
User should call `chatgpt-shell-swap-system-prompt` after calling ` chatgpt-shell-load-awesome-prompts`

### ISSUE THREAD

**Issue by casch-at** [xenodium/chatgpt-shell#226]
https://github.com/xenodium/chatgpt-shell/blob/0efead292f053774bba3c0789370e8e383aa49ec/chatgpt-shell.el#L387-L416

I suggest removing L413-L416, and change the message at L412 to e.g. "Loaded awesome-chatgpt-prompts, call `chatgpt-shell-swap-system-prompt` to change your current `chatgpt-shell-system-prompt`"

**Comment by xenodium** [comment]
Thanks for filing. Could you share a bit more about the use-case? If loading awesome prompts, aren't you already highlighly likely to swap at the time of loading?

**Comment by casch-at** [comment]
Not necessarily, what if the user wants to automatically load them on each Emacs
startup, than the user would get a warning about not being in
`chatgpt-shell-mode`.

**Comment by xenodium** [comment]
> what if the user wants to automatically load them on each Emacs startup, than the user would get a warning about not being in `chatgpt-shell-mode`

If that's the main use-case we'd like to unblock, I'm thinking we should consider making `chatgpt-shell-load-awesome-prompts` a ilttle smarter to determine if invoked interactively to call swap. Would that work for you?

**Comment by casch-at** [comment]
> Would that work for you?

Absolutely 👍🏻 I would be open to implement that feature.

**Comment by xenodium** [comment]
> Absolutely 👍🏻 I would be open to implementing that feature.

Sounds good. `(called-interactively-p #'interactive)` might help here.
